src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h

```diff
@@ -225,6 +228,15 @@ namespace Util
         // string to LayerMsg map
         LayerMsgMap layers_name_to_msg_map;
         ros::Publisher publisher;
+
+        // 15 fps
+        const uint DESIRED_FREQUENCY = 15;
+
+        // Period in nanoseconds
+        const double DESIRED_PERIOD_NS = 1.0e9 / DESIRED_FREQUENCY;
```

Comment:
@MathewMacDougall - thoughts on where constants like this should go?

Reply:
Lets try keep these all in `util/constants.h`